apps/meteor/client/views/omnichannel/currentChats/CurrentChatsRoute.tsx

```diff
@@ -84,19 +101,21 @@ const useQuery: useQueryType = (
 		}
 
 		if (customFields && Object.keys(customFields).length > 0) {
-			query.customFields = JSON.stringify(customFields);
+			const customFieldsQuery: { [key: string]: string } = {};
+			Object.keys(customFields).forEach((key) => {
+				if (customFields[key] !== '') {
+					customFieldsQuery[key] = customFields[key];
+				}
+			});
+			Object.keys(customFieldsQuery).length > 0 && (query.customFields = JSON.stringify(customFieldsQuery));
```

Comment:
It's terrible for readability using conditional expressions for assignments.

```suggestion
			if (Object.keys(customFieldsQuery).length > 0) query.customFields = JSON.stringify(customFieldsQuery);
```